Package.swift

```diff
@@ -1,19 +1,26 @@
+// swift-tools-version:4.0
+
 import PackageDescription
 
 let package = Package(
     name: "VaporApp",
-    targets: [
-        Target(name: "App"),
-        Target(name: "Run", dependencies: ["App"]),
+    products: [
+        .library(name: "App", targets: ["App"]),
+        .executable(name: "Run", targets: ["Run"])
     ],
     dependencies: [
-        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 2),
-        .Package(url: "https://github.com/vapor/leaf-provider.git", majorVersion: 1),
+        .package(url: "https://github.com/vapor/vapor.git", .upToNextMajor(from: "2.1.0")),
```

Comment:
This should probably be 2.2.0 as that's the first version with Swift 4 support

Reply:
It resolves to a working version, not to mention that I copied it straight from the API template. But sure, I'll bump it up.